inc/metadata/namespace.php

```diff
@@ -1016,6 +1017,36 @@ function register_contributor_meta() {
 	);
 }
 
+/**
+ * Apply custom sorting for Contributors list table.
+ *
+ * @param $pieces
+ * @param $taxonomies
+ * @param $args
+ * @return array
+ */
+function contributor_meta_custom_sort( $pieces, $taxonomies, $args ) {
+	$order_field_allowed = [
+		Contributors::TAXONOMY . '_description',
+		Contributors::TAXONOMY . '_institution',
+	];
+	global $pagenow;
+
+	if (
+		$pagenow === 'edit-tags.php' &&
+		isset( $args['taxonomy'] ) &&
+		isset( $args['orderby'] ) &&
+		in_array( Contributors::TAXONOMY, $args['taxonomy'], true ) &&
+		in_array( $args['orderby'], $order_field_allowed, true )
+	) {
+		global $wpdb;
+		$pieces['join'] .= ' INNER JOIN ' . $wpdb->termmeta . ' AS tm ON t.term_id = tm.term_id ';
```

Comment:
Maybe we should use a LEFT JOIN here to avoid removing contributors that don't have a field.

Reply:
oh yes, you're right! I will try with LEFT JOIN and that should solve the issue. Thanks for catching it!